apps/documentation/src/components/StoryEmbed.tsx

```diff
@@ -1,7 +1,7 @@
 import type { StoryObj } from '@storybook/react';
 import type { DecoratorFunction } from '@storybook/types';
-import * as Stories from '@voiceflow/chat/stories';
 import { useEffect, useMemo, useState } from 'react';
+const Stories = typeof window !== 'undefined' ? require('@voiceflow/chat/stories') : {};
```

Comment:
Not sure we need this.. ?

Reply:
I think this is so that we don't render server-side?

Reply:
exactly